integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/table/CarbonDescribeFormattedCommand.scala

```diff
@@ -54,8 +54,12 @@ private[sql] case class CarbonDescribeFormattedCommand(
 
     val carbonTable = relation.carbonTable
     val tblProps = carbonTable.getTableInfo.getFactTable.getTableProperties.asScala
+    // If Sort Columns are given and Sort Scope is not given then pass LOCAL_SORT
+    // as the sort scope, else pass NO_SORT
     val sortScope = if (carbonTable.getNumberOfSortColumns == 0) {
       "NO_SORT"
+    } else if (tblProps.get("sort_scope") != "NO_SORT") {
+      "LOCAL_SORT"
```

Comment:
please use "! equalsIgnoreCase" instead of  "!="
can you describe this branch clearly ?

Reply:
Done.

Reply:
@qiuchenjian When there is no sort scope specified by the user and we have sort columns present then it will take LOCAL_SORT.